bullet_train-super_scaffolding/lib/scaffolding/transformer.rb

```diff
@@ -754,7 +742,7 @@ def valid_#{attribute.collection_name}
 
         # https://stackoverflow.com/questions/21582464/is-there-a-ruby-hashto-s-equivalent-for-the-new-hash-syntax
         if field_options.any? || options.any?
-          field_options_key = if ["buttons", "super_select", "options"].include?(attribute.type)
```

Comment:
Super Select is the only partial that uses `html_options` under the hood, whereas buttons and options simply use `options` (all the BT-side invocations are taken from `main`).

Super Select:
- [select API](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-select) and [BT-side invocation](https://github.com/bullet-train-co/bullet_train-core/blob/cdccbdd270c62592458447e9ee0af5550fdb781c/bullet_train-themes/app/views/themes/base/fields/_super_select.html.erb#L46)

Buttons:
- [check_box API](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-check_box) and [BT-side invocation](https://github.com/bullet-train-co/bullet_train-core/blob/cdccbdd270c62592458447e9ee0af5550fdb781c/bullet_train-themes-tailwind_css/app/views/themes/tailwind_css/fields/_buttons.html.erb#L18)
- [radio_button API](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-radio_button) and [BT-side invocation](https://github.com/bullet-train-co/bullet_train-core/blob/cdccbdd270c62592458447e9ee0af5550fdb781c/bullet_train-themes-tailwind_css/app/views/themes/tailwind_css/fields/_buttons.html.erb#L20)

Options:
- [check_box](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-check_box) and [BT-side invocation](https://github.com/bullet-train-co/bullet_train-core/blob/cdccbdd270c62592458447e9ee0af5550fdb781c/bullet_train-themes-tailwind_css/app/views/themes/tailwind_css/fields/_option.html.erb#L23)
- [check_box_tag](https://api.rubyonrails.org/classes/ActionView/Helpers/FormTagHelper.html#method-i-check_box_tag) and [BT-side invocation](https://github.com/bullet-train-co/bullet_train-core/blob/cdccbdd270c62592458447e9ee0af5550fdb781c/bullet_train-themes-tailwind_css/app/views/themes/tailwind_css/fields/_option.html.erb#L29)
- [radio_button](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-radio_button) and [BT-side invocation](https://github.com/bullet-train-co/bullet_train-core/blob/cdccbdd270c62592458447e9ee0af5550fdb781c/bullet_train-themes-tailwind_css/app/views/themes/tailwind_css/fields/_option.html.erb#L41)